middleware/metrics/metrics_test.go

```diff
@@ -132,25 +191,49 @@ func TestClient(t *testing.T) {
 		return "Hello valid", nil
 	}
 
-	_, err := Client()(nextError)(context.Background(), "test:")
+	// init client handler
+	meter := otel.Meter("test_meter")
+	requests, err := DefaultRequestsCounter(meter, DefaultServerRequestsCounterName)
+	if err != nil {
+		t.Errorf("[DefaultRequestsCounter] something went wrong: %v", err)
+		return
+	}
+	seconds, err := DefaultSecondsHistogram(meter, DefaultServerSecondsHistogramName)
```

Comment:
Does the default need to be exposed? Should be the default implementation when not set by the user

Reply:
In the default logic, we cannot assist users in registering the DefaultSecondsHistogramView. Therefore, we set Option default to nil, allowing users to realize the need to register DefaultSecondsHistogramView.